tests/model_registry/conftest.py

```diff
@@ -233,6 +233,11 @@ def updated_dsc_component_state_scope_class(
                 name=dsc_resource.instance.spec.components.modelregistry.registriesNamespace, ensure_exists=True
             )
             namespace.wait_for_status(status=Namespace.Status.ACTIVE)
+        wait_for_pods_running(
```

Comment:
i'd expect dsc to handle this; i.e if you modify dsc, you should wait for it to be updating and  then ready.

Reply:
It currently does not handle it. I see now pods getting spinned up and not in ready state at this point. I am still trying to see if it is a bug, but for automation to run reliably we need it.